manim/camera/three_d_camera.py

```diff
@@ -392,10 +386,7 @@ def get_static_center_func(mobject):
                 self.fixed_orientation_mobjects[submob] = func
 
     def add_fixed_in_frame_mobjects(self, *mobjects):
-        """This method allows the mobject to have a fixed position,
-        even when the camera moves around.
-        E.G If it was passed through this method, at the top of the frame, it
-        will continue to be displayed at the top of the frame.
+        """Allow the mobject to have a fixed position, even when the camera moves around. E.G If it was passed through this method, at the top of the frame, it will continue to be displayed at the top of the frame.
```

Comment:
```suggestion
        """Allow the mobject to have a fixed position, even when the camera moves around. 
        
        E.G If it was passed through this method, at the top of the frame, it will continue to be displayed at the top of the frame.
```